parser/main.go

```
@@ -1,5 +1,120 @@
+			return
+		}
+		fmt.Println("Folder created successfully!")
+	} else {
+		fmt.Println("Folder already exists!")
+	}
+}
+
+func (parser *Parser) WriteToFile(fileCount int, content *string) {
+	f, err := os.Create(TokenFilePath + strconv.Itoa(fileCount) + ".txt")
+	if err != nil {
+		log.Fatalf("open token error :", err)
```

golangci-lint (1.64.8)

printf: github.com/google/syzkaller/pkg/log.Fatalf call has arguments but no formatting directives

(govet)